content/20.treatments.md

```diff
@@ -286,7 +286,20 @@ When these identifiers are followed back to the registry, most trials claim they
 For all of these 15 identifiers, no data uploads or links to publications could be located on the pre-registrations.
 At the very least, an inability to locate primary data means such claims cannot be verified.
 Additionally, citation of unavailable sources cast doubt on policies and research based on the assumption that HCQ is effective in treating COVID-19.
-
+A randomized, open label, non-placebo, trial of 150 COVID-19 patients was conducted parallelly in 16 government designated COVID-19 centers in China to study safety and efficacy of HCQ[@doi.org/10.1101/2020.04.10.20060558].
+The trial studied efficacy of HCQ in conjunction with standard of care (SOC) vs only SOC in 150 infected patients assigned randomly to two respective groups (75 each group). 
+Most of the cases (98%) in this cohort were mild to moderate with average age of patients being 46 years.
```

Comment:
Revise to, Most of the COVID-19 cases (98%)

Reply:
The change is made